modules/javafx.controls/src/main/java/javafx/scene/control/skin/SplitPaneSkin.java

```diff
@@ -216,8 +221,8 @@ public SplitPaneSkin(final SplitPane control) {
             previousSize = horizontal ? sw : sh;
         }
 
-        // If the window is less than the min size we want to resize
-        // proportionally
+        // If the window is less than the min size we want to resize proportionally
+        duringLayout = true;
         double minSize = totalMinSize();
```

Comment:
- setting the flag belongs above the code comment to not disrupt explanation and its target (== minsize)
- I think we don't do formatting (here: change the code comment to a single line)

Reply:
Changed. 
I kept the comment since it is basically just a single line but yes you are right :)